.github/workflows/ci.yaml

```
@@ -176,25 +176,66 @@
+      - name: Run tests
+        run: bun run test
+
   test-summary:
     if: always()
     runs-on: ubuntu-latest
-    needs: [format-check, cairo-build, build-packages, build-examples]
+    needs: [format-check, cairo-build, build-packages, build-examples, test]
     steps:
       - name: Check all jobs
         run: |
           if [[ "${{ needs.format-check.result }}" != "success" || 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)